sdk/openai/Azure.AI.OpenAI/src/Generated/AudioTranscriptionOptions.Serialization.cs

```diff
@@ -193,6 +201,59 @@ internal static AudioTranscriptionOptions DeserializeAudioTranscriptionOptions(J
                 serializedAdditionalRawData);
         }
 
+        private BinaryData SerializeMultipart(ModelReaderWriterOptions options)
+        {
+            using MultipartFormDataRequestContent content = ToMultipartRequestContent();
+            using MemoryStream stream = new MemoryStream();
```

Comment:
We should not create a memory stream to back a potentially arbitrarily large stream in memory -- the application code could run out of memory.

Reply:
This private function is used in Persistent::Write to persistent the data. and we will convert the multipart to BinaryData to persistent store.